src/frontend-samples/volume-query-sample/VolumeQueryApp.tsx

```diff
@@ -1,163 +1,45 @@
-import { InstanceKey } from "@bentley/presentation-common";
-
-/* Going to color elements from three different sections of volume box */
-export enum SectionOfColoring {
-  InsideTheBox = "Inside",
-  OutsideTheBox = "Outside",
-  Overlap = "Overlap",
-}
-
-/* Going to query and show elements only from two different sections of volume box */
-export enum ElementPosition {
-  InsideTheBox = "Inside",
-  Overlap = "Overlap",
-}
-
-export interface SpatialElement {
-  id: string;
-  name: string | undefined;
-  className: string;
-}
-
-export class VolumeQueryApp {
-
-  /* Method for clearing all clips in the viewport */
-  public static clearClips(vp: ScreenViewport) {
-    // Run the ViewClipClearTool and hide the decorators
-    IModelApp.tools.run(ViewClipClearTool.toolId);
-    ViewClipDecorationProvider.create().toggleDecoration(vp);
-  }
-
-  /* Method for adding a new box range around the model's extents */
-  public static addBoxRange = (vp: ScreenViewport, range?: Range3d, isClippingOn?: boolean) => {
-    // Get the range of the model contents.
-    if (range === undefined) {
-      range = vp.view.computeFitRange();
-      // Remove the top half of the z-range so we have smaller box
-      range.high.z = (range.high.z + range.low.z) / 2.0;
-    }
-    // Create a box for the ClipVector.
-    const block: ClipShape = ClipShape.createBlock(range, range.isAlmostZeroZ ? ClipMaskXYZRangePlanes.XAndY : ClipMaskXYZRangePlanes.All, false, false);
-    // Create the ClipVector
-    const clip: ClipVector = ClipVector.createEmpty();
-    // Add the box to the Clipvector and set it in the ScreenViewport.
-    clip.appendReference(block);
-    // Call enableClipVolume to ensure all clip flags are properly set
-    ViewClipTool.enableClipVolume(vp);
-    // Turning off the clipping feature.
-    vp.view.viewFlags.clipVolume = isClippingOn === undefined ? false : isClippingOn;
-    vp.view.setViewClip(clip);
-    VolumeQueryApp.addDecorators(vp);
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { AuthorizationClient, default3DSandboxUi, SampleIModels, useSampleWidget } from "@itwinjs-sandbox";
+import React, { FunctionComponent, useState } from "react";
+import { Viewer } from "@bentley/itwin-viewer-react";
+import { VolumeQueryWidgetProvider } from "./VolumeQueryWidget";
+import { IModelConnection } from "@bentley/imodeljs-frontend";
+import { IModelViewportControlOptions } from "@bentley/ui-framework";
+import { VolumeQueryApi } from "./VolumeQueryApi";
+
+const uiProviders = [new VolumeQueryWidgetProvider()];
+
+const VolumeQueryApp: FunctionComponent = () => {
+  const sampleIModelInfo = useSampleWidget("Use the controls below to change the view attributes.", [SampleIModels.RetailBuilding]);
```

Comment:
```suggestion
  const sampleIModelInfo = useSampleWidget("Use the controls below to query elements inside, overlapping, and outside of the clip.", [SampleIModels.RetailBuilding]);
```

... or similar

Reply:
Use the controls below to query and color spatial elements in the iModel using a volume box.

Reply:
Fixed.